src/Aspire.Hosting/Dashboard/ResourceSnapshot.cs

```diff
@@ -22,8 +23,8 @@ internal abstract class ResourceSnapshot
     public required DateTime? CreationTimeStamp { get; init; }
     public required ImmutableArray<EnvironmentVariableSnapshot> Environment { get; init; }
     public required ImmutableArray<VolumeSnapshot> Volumes { get; init; }
-
     public required ImmutableArray<UrlSnapshot> Urls { get; init; }
+    public HealthStateKind HealthState { get; set; }
```

Comment:
This type's immutable, so this'd be better as `init`. Could also be `required` for consistency with the other properties.